### DISCUSSION TITLE
Multiple requests opened like postman

### DISCUSSION THREAD

**Discussion by arthurvanl** [Kong/insomnia#7847] [Ideas]
![image](https://github.com/user-attachments/assets/ddb676f7-c393-4d90-92e4-03baf60c0fb9)

In postman you can have multiple requests opent in different tabs. Would be very usefull if this added into insomnia!

**Comment by subnetmarco** [comment]
Yes, we are planning to ship it in the October timeframe!

**Comment by arthurvanl** [comment]
Awesome! Thanks for the fast response!

**Comment by arthurvanl** [comment]
Is there a beta version, so I can check it out?

**Reply by arthurvanl** [reply]
already found it

**Reply by arthurvanl** [reply]
Looks like it does not exist yet

**Reply by subnetmarco** [reply]
The current beta doesn't include it.
